### DISCUSSION TITLE
Disable system tables activity log?

### DISCUSSION THREAD

**Discussion by JonL1** [directus/directus#12398] [Archive]
Is it possible?

**Comment by rijkvanzanten** [comment]
You can disable activity tracking through Data Model > Collection > Accountability > Activity & Revisions Tracking 👍🏻

**Reply by JonL1** [reply]
I was referring to Directus system collections. I believe these are hard coded but I was wondering if there was some configuration flag.

**Reply by rijkvanzanten** [reply]
Gotcha, no those can't be disabled currently 👍🏻
